src/core/server/legacy/legacy_service.ts

```diff
@@ -275,6 +275,8 @@ export class LegacyService implements CoreService {
         addClientWrapper: setupDeps.core.savedObjects.addClientWrapper,
         registerType: setupDeps.core.savedObjects.registerType,
         getImportExportObjectLimit: setupDeps.core.savedObjects.getImportExportObjectLimit,
+        registerRepositoryFactoryProvider:
```

Comment:
lets name this function as `setRepositoryFactoryProvider` to be consistent as setting client factory provider function

Reply:
Changed the function name.